peer.go

```diff
@@ -363,13 +371,22 @@ out:
 			break out
 		}
 
+		// Should wait for peers to compare their feature vectors
+		// and only then start message exchanges.
+		if !initialized {
+			if msg, ok := nextMsg.(*lnwire.Init); ok {
+				p.handleInitMsg(msg)
+				initialized = true
+			}
+			continue
```

Comment:
I think it would be better to stay synced with specification, and as far as we have `alpha` release it shouldn't be an issue. So with your approve I would like to choice `backwards incompatible change`.